backend/api/display.go

```diff
@@ -7,7 +7,15 @@ import (
 
 func GameWindow(client *Client, event *Event) GameError {
 	session := strings.Split(event.Session, ":")
-	roomCode, _ := session[0], session[1]
+	if len(session) < 2 {
+		return GameError{code: "INVALID_SESSION_FORMAT", message: "Session must be in format 'roomCode:id' or 'roomCode:id:0' for spectators"}
+	}
+
+	roomCode := session[0]
+	if roomCode == "" {
+		return GameError{code: "INVALID_ROOM", message: "Room code cannot be empty"}
```

Comment:
The `code` in `GameError` actually matches the errors dot notation in the translation files.

Check out https://github.com/karlromets/Cold-Family-Feud/blob/dc8ce4f33ba55149188f3cde6031f7e3e4420c92/backend/api/errors.go#L11

For available variables you can use.

You only want to put in a `message` if the translation requires a message.